File: ClientApp/src/app/models/state.ts


export class State {
  constructor(
    public authenticated: boolean = false,
    public overviewMode: boolean = false,
    public currentQuranSura: number = 1,
    public currentTafseerAndTranSura: number = 1,
    public currentTafseerAndTranAya: number = 1,
    public currentNoteSura: number = 1,
    public currentNoteAya: number = 1,
    public currentSearch: string = "",
    public searchForWord: boolean = false,
    public showLetters: boolean = false,
    public currentStatisticsPage:number = 1,
    public currentMindMapSura: number = 1  ) { }
}

File: ServerApp/Models/State.cs
﻿namespace ServerApp.Models
{
    public class State
    {
        public bool authenticated {get; set;}
        public bool overviewMode {get ; set;}
        public int currentQuranSura { get; set; }
        public int currentTafseerAndTranSura {get; set; }
        public int currentTafseerAndTranAya  {get; set; }
        public int currentNoteSura {get; set; }
        public int currentNoteAya  {get; set; }
        public string currentSearch { get; set; }
        public bool searchForWord { get; set; }
        public bool showLetters {get; set;}
        public int currentStatisticsPage { get; set; }
    }


}
